rasa_nlu/classifiers/intent_threshold.py

```diff
@@ -0,0 +1,76 @@
+from __future__ import unicode_literals
+from __future__ import print_function
+from __future__ import division
+from __future__ import absolute_import
+
+import os
+import io
+import json
+from builtins import map
+from typing import Any
+from typing import Dict
+from typing import Text
+import re
+
+
+import rasa_nlu.converters as converters
+from rasa_nlu.training_data import TrainingData
+
+from rasa_nlu.components import Component
+
+
+class IntentThreshold(Component):
```

Comment:
Name needs to mention out of scope - otherwise people will think this is an intent classifier for all intents.